councilmatic_core/management/commands/import_data.py

```diff
@@ -974,7 +974,8 @@ def insert_raw_bills(self, delete=False):
             else:
                 raise Exception(bill_info['classification'])
 
-            slug = slugify(bill_info['identifier'])
```

Comment:
This seems like a pretty disruptive change to all the other councilmatics. Why can't we have this be something that we can set in settings.py file?

Reply:
As I mentioned in the checkin, this PR is entirely unnecessary. I'll close it.